crates/bitwarden-crypto/src/store/context.rs

```diff
@@ -167,9 +164,10 @@ impl<Ids: KeyIds> KeyStoreContext<'_, Ids> {
         &self,
```

Comment:
We should probably consider renaming the function `wrap_symmetric_key` or something like that as well